### PR TITLE
added tests to test_to_hex and test_to_hex_trimmed

### LINKED ISSUE #322: Add tests for the to_hex and the to_hex_trimmed
## Description 📹

**Add tests for the "to_hex" and the "to_hex_trimmed" functions (from the utils folder) like we’ve done in `tests/endpoints.rs`**

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
- [ ] Create a "utils.rs" file in the `tests` folder
- [ ] Fill it with the two requested tests
- [ ] You can reuse the way we've done tests in `tests/endpoints.rs*`

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by guha-rahul**
close #322

### CODE REVIEW

File: tests/utils.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#[cfg(test)]
+mod tests {
+    use crate::utils::{to_hex, to_hex_trimmed};
```
Comment (guha-rahul) [inline tests/utils.rs]: had some trouble importing the functions. would require some help on this

**Comment by Marchand-Nicolas** [comment]
@guha-rahul  In fact, (I didn't sorry), the convention is to put the tests in the same file where the function you are testing is wrote https://doc.rust-lang.org/book/ch11-03-test-organization.html#:~:text=You'll%20put%20unit%20tests,module%20with%20cfg(test)%20.

So you can directly put them there

**Comment by Marchand-Nicolas** [comment]
Ah @guha-rahul this dev found a good solution, you can check it: https://github.com/lfglabs-dev/api.starknet.id/pull/112
I think it's better than having the tests in the same file (because utils.rs is already huge)

**Comment by Marchand-Nicolas** [comment]
Please resolve the conflicts so that I can merge

File: src/tests/mod.rs [hunk 2]
```diff
@@ -0,0 +1 @@
+mod utils;
```
Comment (Marchand-Nicolas) [inline src/tests/mod.rs:1]: Also add mod endpoints

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

File: src/main.rs [hunk 3]
```diff
@@ -24,6 +24,23 @@ lazy_static::lazy_static! {
...
+    log: LogPayload<'a>,
+}
+
+#[derive(Serialize)]
+struct LogPayload<'a> {
+    app_id: &'a str,
+    r#type: &'a str,
+    message: Cow<'a, str>,
+    timestamp: i64,
+}
```
Comment (Marchand-Nicolas) [inline src/main.rs]: This is not needed, and it doesn't build.
I think you can just keep

```rs
#[cfg(test)]
mod tests;
```

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

**Review by Marchand-Nicolas — APPROVED** [review]
Well done lgtm!

**Pull request merged**